CHANGELOG.md

```diff
@@ -7,6 +7,9 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [Unreleased]
 
+### Changed 
+- [#847](https://github.com/FuelLabs/fuel-vm/pull/847): Remove `contract_read`, and change `load_contract_code`, `code_copy` and `code_root`  to explicitly load the contract code in a buffer. Also check for mismatches between contract size stored and actual size of contract in those functions.
```

Comment:
I'm confused by this. There is no function called `contract_read` that is removed in these changes. And you added a call to `read_contract`, which sounds like the same thing?